app/javascript/components/patient/laboratory/LaboratoryModal.js

```diff
@@ -148,7 +148,7 @@ class LaboratoryModal extends React.Component {
                     id="symptom_onset_lab"
                     date={this.state.symptom_onset}
                     minDate={'2020-01-01'}
-                    maxDate={moment().add(30, 'days').format('YYYY-MM-DD')}
+                    maxDate={moment().format('YYYY-MM-DD')}
```

Comment:
Since this sort of check is not in `Exposure.js`, when enrolling someone, you can select a SO in the future, and then you get a validation error about it, where here you just don't even have the option to select a date in the future. I'm fine with the validation error when enrolling, since I think that is consistent with how other errors are shown, but it also may be simpler to just not allow a date in the future at all. So I wanted to put it to you, which do you think makes more sense?

Reply:
oops, good catch! https://github.com/SaraAlert/SaraAlert/pull/1018/commits/c971496ea442bb6c9e53fb78384ddc7b0dbbec03